doc/source/deployment/configure.rst

```diff
@@ -190,6 +190,16 @@ For example:
 
    socok8s_ext_vip: "10.10.10.10"
 
+Set Up Retry Files Save Path
+----------------------------
+
+Before beginning deployment, a path must be specified where Ansible retry files
+can be saved in order to avoid potential errors. The path must point to a
+user-writable directory. Set the path in either of the following ways:
+
+- ``export ANSIBLE_RETRY_FILES_SAVE_PATH=<PATH_TO_FILE>`` before deploying with
+  ``run.sh`` commands.
+- Set the value of `retry_files_save_path` in your Ansible configuration file.
```

Comment:
There's actually also an option to disable creating those retry files. Use `retry_files_enabled = False` setting in ansible conf file for that